### PR TITLE
build: setup boilerplate

### PR DESCRIPTION
**Pull request by lorenzogm**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "endOfLine": "lf",
+  "printWidth": 120,
+  "semi": false,
```
Comment (b12k) [inline .prettierrc:6]: Why `semi: false`?

File: .releaserc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (b12k) [inline .releaserc:1]: How do you expect Semantic release to work inside of monorepo?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+.pnpm-debug.log*
+
+# local env files
+.env
```
Comment (b12k) [inline .gitignore:27]: There are no other `.env` then `.env`. Its used **only** for development.

File: .prettierignore [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.idea
```
Comment (b12k) [inline .prettierignore:1]: this is actually a .gitignore

File: .prettierrc (hunk 1 above)
Comment (lorenzogm) [reply .prettierrc:6]: Just used to it, they are not needed. Any reason why is better to keep them?

File: .releaserc (hunk 2 above)
Comment (lorenzogm) [reply .releaserc:1]: Just copy-paste, better if I remove it and we think what can be done. I just found this: https://github.com/pmowrer/semantic-release-monorepo

File: .prettierignore (hunk 4 above)
Comment (lorenzogm) [reply .prettierignore]: updated

File: .gitignore (hunk 3 above)
Comment (lorenzogm) [reply .gitignore:27]: those were the default, I guess they don't hurt to have them there

File: .prettierrc (hunk 1 above)
Comment (b12k) [reply .prettierrc:6]: Well, consistency? =)
If you are working in multilang project:
- scss - required
- css - required
- php - required
- c# - required
- java - required

**Review by b12k — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (lorenzogm) [reply .prettierrc:6]: good point, so I'm happy to enable them

**Pull request merged**